tests/utils/configuration/test_configurable.py

```diff
@@ -1,84 +0,0 @@
-from dataclasses import dataclass
```

Comment:
Would be good to unit test the deserialiszation conversion, even if it's not critical to the running of tickit